README.md

```diff
@@ -1 +1,40 @@
-# terraform-azurerm-rg
+<!-- BEGIN_TF_DOCS -->
```

Comment:
please check and include all automated check, validate tools we have(like checkov, terraform test, terrform fmt, terrform docs ....), see terraform_aws_modules or terraform_aws_repository,
you even can use terraform_aws_repository to automatically add this all

Reply:
done